### PR TITLE
addition of 2023C WFs and modifications 2023B WFs to run with era Run3

### PR DESCRIPTION
**Pull request by sunilUIET**
This PR is to add WFs for 2023C and modify 2023B WFs to use era Run3 (in place of era Run3_2023). While implementation, https://github.com/cms-sw/cmssw/pull/42031 is taken into account to add rerecoZeroBias DQM for ZeroBias PD. 
This PR new WFs 141.031-141.038

Once merged in the master, these changes will be backported to 13_0_X

### CODE REVIEW

**Comment by sunilUIET** [comment]
test parameters:

relvals_opt= -w standard
workflows = 140.001,140.002,140.003,140.004,140.005,140.006,140.007,140.008,140.009,140.010,140.011,140.012,140.013,140.031,140.032,140.033,140.034,140.035,140.036,140.037,140.038

**Comment by sunilUIET** [comment]
Hi @mmusich 

I had to make some changes for 2023B WFs which conflicted with your PR https://github.com/cms-sw/cmssw/pull/42031

Please have a look.

**Comment by sunilUIET** [comment]
test parameters:

relvals_opt= -w standard
workflows = 141.001,141.002,141.003,141.004,141.005,141.006,141.007,141.008,141.009,141.010,141.011,141.012,141.013,141.031,141.032,141.033,141.034,141.035,141.036,141.037,141.038

**Comment by mmusich** [comment]
@sunilUIET 

> I had to make some changes for 2023B WFs which conflicted with your PR https://github.com/cms-sw/cmssw/pull/42031

I don't see conflicts.

**Comment by sunilUIET** [comment]
> @sunilUIET
> 
> > I had to make some changes for 2023B WFs which conflicted with your PR #42031
> 
> I don't see conflicts.

Yes, I fixed it now.

**Comment by mmusich** [comment]
> Yes, I fixed it now.

so, I am not sure what I should comment about. I guess we should coordinate for the backports instead.

**Comment by sunilUIET** [comment]
please test

**Comment by sunilUIET** [comment]
> > Yes, I fixed it now.
> 
> so, I am not sure what I should comment about. I guess we should coordinate for the backports instead.

Yes, as we discussed, I will take care of backporting these changes keeping in mind the changes coming along with backports of https://github.com/cms-sw/cmssw/pull/42069

**Comment by sunilUIET** [comment]
+pdmv

File: Configuration/PyReleaseValidation/python/relval_standard.py [hunk 1]
```diff
@@ -510,19 +510,32 @@
...
-workflows[141.005] = ['',['RunHcalNZS2023B','HLTDR3_2023','RECONANORUN3_reHLT_2023','HARVESTRUN3_2023']]
-workflows[141.006] = ['',['RunHLTPhysics2023B','HLTDR3_2023','RECONANORUN3_reHLT_2023','HARVESTRUN3_2023']]
-workflows[141.007] = ['',['RunCommissioning2023B','HLTDR3_2023','RECONANORUN3_reHLT_2023','HARVESTRUN3_2023']]
-workflows[141.008] = ['',['RunJetMET2023B','HLTDR3_2023','RECONANORUN3_reHLT_2023','HARVESTRUN3_2023']]
-workflows[141.009] = ['',['RunCosmics2023B','HLTDR3_2023','RECONANORUN3_reHLT_2023','HARVESTRUN3_2023']]
-workflows[141.010] = ['',['RunDisplacedJet2023B','HLTDR3_2023','RECONANORUN3_reHLT_2023','HARVESTRUN3_2023']]
-workflows[141.011] = ['',['RunEGamma2023B','HLTDR3_2023','RECONANORUN3_reHLT_2023','HARVESTRUN3_2023']]
-workflows[141.012] = ['',['RunTau2023B','HLTDR3_2023','RECONANORUN3_reHLT_2023','HARVESTRUN3_2023']]
-workflows[141.013] = ['',['RunMuonEG2023B','HLTDR3_2023','RECONANORUN3_reHLT_2023','HARVESTRUN3_2023']]
+
```
Comment (srimanob) [inline Configuration/PyReleaseValidation/python/relval_standard.py:513]: need space here?

Comment (sunilUIET) [reply Configuration/PyReleaseValidation/python/relval_standard.py:513]: Hi @srimanob 

   IIUC, you are talking additional empty line between 2023B and 2023C WFs. This is not part of the planned changes and was introduced accidentally.

**Comment by srimanob** [comment]
test parameters:

relvals_opt= -w standard
workflows = 141.001,141.002,141.003,141.004,141.005,141.006,141.007,141.008,141.009,141.010,141.011,141.012,141.013,141.031,141.032,141.033,141.034,141.035,141.036,141.037,141.038
pull_request = https://github.com/cms-sw/cmssw/pull/42047

**Comment by srimanob** [comment]
@cmsbuild please test

**Comment by sunilUIET** [comment]
test parameters:

relvals_opt= -w standard
workflows = 141.001,141.002,141.003,141.004,141.005,141.006,141.007,141.008,141.009,141.010,141.011,141.012,141.013,141.031,141.032,141.033,141.034,141.035,141.036,141.037,141.038
pull_request = https://github.com/cms-sw/cmssw/pull/42047

**Comment by sunilUIET** [comment]
@cmsbuild please test

**Comment by srimanob** [comment]
Hi @sunilUIET 
Thanks for pinging.

`steps['RECONANORUN3_ZB_reHLT_2022']` should use `steps['RECONANORUN3_reHLT_2022']` as the base, not `steps['RECODR3_reHLT_2022']`

Could you please include the fix in the following lines,
https://github.com/cms-sw/cmssw/blob/master/Configuration/PyReleaseValidation/python/relval_steps.py#L2890
and
https://github.com/cms-sw/cmssw/blob/master/Configuration/PyReleaseValidation/python/relval_steps.py#L2894

Thx.

**Comment by sunilUIET** [comment]
> Hi @sunilUIET Thanks for pinging.
> 
> `steps['RECONANORUN3_ZB_reHLT_2022']` should use `steps['RECONANORUN3_reHLT_2022']` as the base, not `steps['RECODR3_reHLT_2022']`
> 
> Could you please include the fix in the following lines, https://github.com/cms-sw/cmssw/blob/master/Configuration/PyReleaseValidation/python/relval_steps.py#L2890 and https://github.com/cms-sw/cmssw/blob/master/Configuration/PyReleaseValidation/python/relval_steps.py#L2894
> 
> Thx.

Yes, I will take care of it

**Comment by sunilUIET** [comment]
test parameters:

relvals_opt= -w standard
workflows = 141.001,141.002,141.003,141.004,141.005,141.006,141.007,141.008,141.009,141.010,141.011,141.012,141.013,141.031,141.032,141.033,141.034,141.035,141.036,141.037,141.038

**Comment by sunilUIET** [comment]
@cmsbuild please test

**Comment by sunilUIET** [comment]
+pdmv

**Comment by srimanob** [comment]
+Upgrade

**Pull request merged**